### PR TITLE
Website amends

### PR DESCRIPTION
**Pull request by Guntrisoft**
screen shot 2016-12-06 at 13 53 19

### CODE REVIEW

File: app/views/appointment_summaries/new.html.erb [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<% content_for(:page_title, 'How to view your appointment summary - Pension Wise') %>
```
Comment (benbarnett) [inline app/views/appointment_summaries/new.html.erb]: Can we move these into the locale file as per the other pages?

Also just noticed that the F2F booking pages are all missing page titles too!

Comment (Guntrisoft) [inline app/views/appointment_summaries/new.html.erb]: I've used the t('service.title', X) pattern for these missing page titles - gone through hopefully all pages that are missing a title and added one.

**Comment by Guntrisoft** [comment]
@alexkouz have a 👀

File: app/views/booking_requests/step_two.html.erb [hunk 2]
```diff
@@ -1,3 +1,4 @@
+<% content_for(:page_title, t('service.title', page_title: "Your details")) %>
```
Comment (benlovell) [inline app/views/booking_requests/step_two.html.erb]: Quotes.

Comment (Guntrisoft) [inline app/views/booking_requests/step_two.html.erb]: Damnation

**Pull request merged**